cpp/oneapi/dal/table/test/csr_accessor.cpp

```diff
@@ -16,11 +16,35 @@
 
 #include "oneapi/dal/test/engine/common.hpp"
 #include "oneapi/dal/test/engine/fixtures.hpp"
+#include "oneapi/dal/test/engine/linalg.hpp"
 #include "oneapi/dal/table/csr_accessor.hpp"
 
+#include <iostream>
```

Comment:
Is it required to include iostream here?

Reply:
No, thank you for noticing. This line was added due to debug printing and should be removed.